lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -535,53 +535,98 @@ CRI_HMRC_KBV_J6:
     next:
       targetState: PYI_NO_MATCH
 
-# No photo id M2B journey (J7)
-CRI_CLAIMED_IDENTITY_J7:
+# No photo id journey (M2B)
+CRI_CLAIMED_IDENTITY_M2B:
```

Comment:
(You probably already know, but in general renaming states needs a two-step deployment - no need in this case because it's not live :) )